packages/ember-htmlbars/tests/helpers/unbound_test.js

```diff
@@ -1,595 +0,0 @@
-      },
-      compute([value]) {
-        if (this.get('value')) {
-          this.teardown();
-        }
-        this.set('value', value);
-        this.addObserver('value.firstName', this, this.recompute);
-        this.addObserver('value.lastName', this, this.recompute);
-        return (value ? get(value, 'firstName') : '') + (value ? get(value, 'lastName') : '');
-      }
-    }));
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-    owner = view = null;
-    Ember.lookup = originalLookup;
-  }
-});
-
-QUnit.test('should be able to render an unbound helper invocation', function() {
-  owner.register('helper:-repeat', helper(function([value], { count }) {
-    var a = [];
-    while (a.length < count) {
-      a.push(value);
-    }
-    return a.join('');
-  }));
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{unbound (-repeat foo count=bar)}} {{-repeat foo count=bar}} {{unbound (-repeat foo count=2)}} {{-repeat foo count=4}}'),
-    context: EmberObject.create({
-      foo: 'X',
-      numRepeatsBinding: 'bar',
-      bar: 5
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'XXXXX XXXXX XX XXXX', 'first render is correct');
-
-  run(function() {
-    set(view, 'context.bar', 1);
-  });
-
-  equal(view.$().text(), 'XXXXX X XX XXXX', 'only unbound bound options changed');
-});
-
-QUnit.test('should be able to render an bound helper invocation mixed with static values', function() {
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{unbound (-surround prefix value "bar")}} {{-surround prefix value "bar"}} {{unbound (-surround "bar" value suffix)}} {{-surround "bar" value suffix}}'),
-    context: EmberObject.create({
-      prefix: 'before',
-      value: 'core',
-      suffix: 'after'
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'before-core-bar before-core-bar bar-core-after bar-core-after', 'first render is correct');
-  run(function() {
-    set(view, 'context.prefix', 'beforeChanged');
-    set(view, 'context.value', 'coreChanged');
-    set(view, 'context.suffix', 'afterChanged');
-  });
-  equal(view.$().text(), 'before-core-bar beforeChanged-coreChanged-bar bar-core-after bar-coreChanged-afterChanged', 'only bound values change');
-});
-
-QUnit.test('should be able to render unbound forms of multi-arg helpers', function() {
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{-fauxconcat foo bar bing}} {{unbound (-fauxconcat foo bar bing)}}'),
-    context: EmberObject.create({
-      foo: 'a',
-      bar: 'b',
-      bing: 'c'
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'abc abc', 'first render is correct');
-
-  run(function() {
-    set(view, 'context.bar', 'X');
-  });
-
-  equal(view.$().text(), 'aXc abc', 'unbound helpers/properties stayed the same');
-});
-
-QUnit.test('should be able to render an unbound helper invocation for helpers with dependent keys', function() {
```

Comment:
:heavy_check_mark: should be able to render an unbound helper invocation for helpers with dependent keys